dev/com.ibm.ws.jaxrs.2.1_fat_executor/bnd.bnd

```diff
@@ -26,4 +26,7 @@ javac.target: 1.8
   com.ibm.websphere.javaee.jsonb.1.0,\
   com.ibm.websphere.javaee.jsonp.1.1,\
   com.ibm.websphere.javaee.servlet.3.1;version=latest,\
-  com.ibm.ws.cdi.interfaces;version=latest
+  com.ibm.ws.cdi.interfaces;version=latest,\
+  com.ibm.websphere.org.eclipse.microprofile.contextpropagation.1.0;version=latest,\
+  com.ibm.ws.concurrent.mp.1.0;version=latest,\
+  com.ibm.websphere.javaee.concurrent.1.0;version=latest  
```

Comment:
I think only `com.ibm.websphere.javaee.concurrent.1.0;version=latest` is necessary here.  The test cases shouldn't need to depend on MP context propagation or the concurrent.mp project.  If they do, then it sounds like user apps might also need these dependencies, which would be a breaking change.